### PR TITLE
Disabled the ability to butcher/disassemble tinder.

### LINKED ISSUE #23525: Tinder can be cut into pieces of wood
Game version: 0.C-27399-ga98c6b1

Operating system: Windows 7

Tiles or curses: Tiles

Mods active: More Survival Tools, No Explosive Zombies, No Fungal Monsters, StatsThroughSkill IV, Vehicle Additions Pack, Tanks and Other Vehicles, Bionics Systems Mod, Boats, Icecoon's Arsenal, Filthy Clothing, Makeshift Items Mod, More Buildings, No Flaming Weapons, Disable Mutgens, Disable NPC Needs, No Antique Firearms, No Fictional Guns, NPC Traits

### Expected behavior
Tinder cannot be cut through 'B'utchering menu

### Actual behavior
Tinder can be 'B'utchered into pieces of splintered wood

### Steps to reproduce the behavior
'B'utcher tinder

### PR DESCRIPTION
**Pull request by 0Petya**
This closes #23525.

### CODE REVIEW

**Comment by kevingranade** [comment]
The issue is just that the json isn't formatted correctly, there should be spaces between the [] brackets and their contents.

File: data/mods/More_Survival_Tools/ammo.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "price": 0,
     "bashing": 1,
     "material": "wood",
+    "flags" : ["NO_SALVAGE"],
```
Comment (ZhilkinSerg) [inline data/mods/More_Survival_Tools/ammo.json]:
```json
"flags": [ "NO_SALVAGE" ],
```

**Comment by 0Petya** [comment]
Cool, it looks like @ZhilkinSerg's change should fix that.

Was there a way for me to tell that that was the problem with the CI build?

**Comment by ZhilkinSerg** [comment]
You can click on red X and look for text of the error.

**Comment by 0Petya** [comment]
Yeah I looked at the log and can see the error, I just didn't understand how to know that the error meant that my formatting was wrong.

**Comment by ZhilkinSerg** [comment]
When you see these errors - that means JSON formatting is broken.

# Jenkins

```
07:50:50 Formatted data/mods/More_Survival_Tools/ammo.json
07:50:50 Makefile:992: recipe for target 'data/mods/More_Survival_Tools/ammo.json' failed
07:50:50 make: *** [data/mods/More_Survival_Tools/ammo.json] Error 1
```

# Travis CI

```
Formatted data/mods/More_Survival_Tools/ammo.json
make: *** [data/mods/More_Survival_Tools/ammo.json] Error 1
```

**Comment by 0Petya** [comment]
Oh ok, thanks!

**Pull request merged**